cmake/usMacroCreateBundle.cmake

```diff
@@ -43,6 +43,22 @@ if(${PROJECT_NAME}_DEPENDS)
   )
 endif()
 
+# Set the resource embedding method
+set(_resource_embed_type )
+if(${PROJECT_NAME}_LINK_RESOURCES AND ${PROJECT_NAME}_APPEND_RESOURCES)
+  message(SEND_ERROR "Both APPEND and LINK options were specified. These options are mutually exclusive.")
+endif()
+
+if(${PROJECT_NAME}_LINK_RESOURCES)
+  set(_resource_embed_type LINK)
+  if(NOT ${US_RESOURCE_LINKING_AVAILABLE} OR ${US_RESOURCE_LINKING_AVAILABLE} EQUAL 0)
+    message(WARNING "Resource linking was specified however its not supported on the system. Defaulting to ${US_DEFAULT_RESOURCE_MODE}.")
+    set(_resource_embed_type ${US_DEFAULT_RESOURCE_MODE})
+  endif()
```

Comment:
this can be removed. It is a redundant check which is already performed in `usFunctionEmbedResources.cmake`